README.md

```diff
@@ -1,4 +1,4 @@
-# Marathon UI [![Build Status](https://travis-ci.org/mesosphere/marathon-ui.png?branch=master)](https://travis-ci.org/mesosphere/marathon-ui)
+# Marathon UI [![Build Status](https://travis-ci.org/mesosphere/marathon-ui.png?branch=master)](https://travis-ci.org/mesosphere/marathon-ui) [![Stories in Ready](https://badge.waffle.io/mesosphere/marathon.png?label=read,gui&title=Ready)](https://waffle.io/mesosphere/marathon)
```

Comment:
I still think this needs some explanation. It is not clear to me what "Ready" and a number means.